src/app/ecosystem-explorer/project-form/page.tsx

```diff
@@ -0,0 +1,78 @@
+'use client'
+
+import { useState } from 'react'
+
+import { Button } from '@/components/Button'
+
+import { submitProjectToGithub } from './services/github'
+import {
+  ALLOWED_IMAGE_FORMATS,
+  getFileFormat,
+  convertToBase64,
+} from './services/github/utils'
```

Comment:
I think this is one of those cases - Do we use `@` or?

Reply:
I don't understand. Do you want to use `@` here?

Reply:
I don't know - I thought that's what you said on a call last time?